.github/workflows/post-pull-request-checks-automation.yml

```diff
@@ -40,26 +40,49 @@ on:
         type: number
         default: 1
         description: Configuration for the timeout-minutes parameter of the workflow.
+      run-only-latest-workflow:
+        type: string
+        description: 'Run only if the current workflow is the most recent workflow.'
+        default: 'true'
 
 jobs:
   post-pull-request-checks-automation:
+    outputs:
+      latest-run-number: ${{ steps.get-latest-workflow-run-number.outputs.latest_run_number }}
     runs-on: ubuntu-22.04
     timeout-minutes: ${{ inputs.timeout-minutes }}
     steps:
+      - name: Checkout
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get Latest Workflow Run Number
+        id: get-latest-workflow-run-number
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          LATEST_RUN_NUMBER=$(gh run list --limit 1 --json databaseId | jq -r '.[0].databaseId')
+          echo "latest_run_number=$LATEST_RUN_NUMBER" >> $GITHUB_OUTPUT
```

Comment:
Still https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2545614855/Writing+GitHub+Actions+workflows+and+actions#:~:text=Prefer%20to%20use%20PowerShell%207%2B%20scripts